aten/src/ATen/native/quantized/cpu/qnnpack/src/qnnpack/requantization.h

```diff
@@ -159,12 +159,12 @@ pytorch_qnnp_compute_conv_quantization_params(
 #if CPUINFO_ARCH_X86 || CPUINFO_ARCH_X86_64
   for (uint32_t i = 0; i < 8; i++) {
     params.sse2.input_zero_point[i] = (int16_t)(uint16_t)input_zero_point;
-    params.sse2.kernel_zero_point[i] = (int16_t)(uint16_t)kernel_zero_point;
+    params.sse2.kernel_zero_point[i] = (int16_t)(uint16_t)kernel_zero_points[0];
```

Comment:
Again I think these casts are not necessary.

Reply:
Again all these exist here from earlier. Dont think these set of PRs is the right ones to fix it.